### PR TITLE
Add Python 3.6 configuration in travis and tox.

### PR DESCRIPTION
**Pull request by Natim**
Refs #210

- [x] Tests
- [x] Changelog amended with a mention describing your changes.

### CODE REVIEW

**Comment by brunobord** [comment]
I think you're clear to squash, I'll merge it when I can

File: CHANGELOG [hunk 1]
```diff
@@ -4,7 +4,8 @@ CHANGELOG
 master (unreleased)
 -------------------
 
-Nothing here yet.
+- Run tests with Python3.6 in CI
```
Comment (brunobord) [inline CHANGELOG]: Could you please refer to the issue number here?

**Comment by Natim** [comment]
You should be able to use the Github button to merge and squash.
For some reason described [here](http://tech.novapost.fr/git-history-matters-en.html) and [there](http://tech.novapost.fr/git-history-matters-example-en.html) I am not fan of squashing because it removes the history of what happened and it makes people think that everything was right the first time which is not the case.

**Comment by brunobord** [comment]
> I am not fan of squashing

/me shrugs
No strong opinion on this, don't worry, I'm still very excited that we're Python 3.6 ready!

**Pull request merged**